src/util/animate.js

```diff
@@ -47,7 +52,7 @@
             timePerc = currentTime / duration,
             current = easing(currentTime, startValue, byValue, duration),
             valuePerc = Math.abs((current - startValue) / byValue);
-        if (abort()) {
+        if (aborted || abort()) {
```

Comment:
let's divide it as it was here:
https://github.com/fabricjs/fabric.js/pull/5936/files#diff-4c788168c81ce977ee91d228587e70e438fdd375b832462bcc998fa1e111ff4aR55

So that then the old abort() can be deprecated easily.

Reply:
Don't you want to support both imperative and declarative abort?